tutorials/LinearAlgebra/LinearAlgebra.ipynb

```diff
@@ -0,0 +1,983 @@
+    "    a \\cdot x_{1,0} & a \\cdot x_{1,1} & \\dotsb & a \\cdot x_{1,m-1} \\\\\n",
+    "    \\vdots  & \\vdots  & \\ddots & \\vdots  \\\\\n",
+    "    a \\cdot x_{n-1,0} & a \\cdot x_{n-1,1} & \\dotsb & a \\cdot x_{n-1,m-1}\n",
+    "\\end{bmatrix}$$\n",
+    "\n",
+    "The scalar may be real or complex.\n",
+    "\n",
+    "Scalar multiplication has the following properties:\n",
+    "\n",
+    "* Associativity: $x \\cdot (yA) = (x \\cdot y)A$\n",
+    "* Distributivity over matrix addition: $x(A + B) = xA + xB$\n",
+    "* Distributivity over scalar addition: $(x + y)A = xA + yA$"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <span style=\"color:blue\">Exercise 2</span>: Scalar multiplication\n",
+    "\n",
+    "**Inputs:**\n",
+    "\n",
+    "1. A scalar $x$.\n",
+    "2. An $n \\times m$ matrix $A$.\n",
+    "\n",
+    "**Goal:** Return the matrix that results from the multiplication $x \\cdot A$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "@exercise\n",
+    "def scalar_mult(x : complex, a : Matrix) -> Matrix:\n",
+    "    # Fill in the missing code and run the cell to check your work.\n",
+    "    return # ..."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Matrix Multiplication\n",
+    "\n",
+    "A more complicated, but much more important and interesting operation is **matrix multiplication**. The first complicated thing about it is that neither its operands nor its output are the same size: an $n \\times m$ matrix multiplied by an $m \\times k$ matrix results in an $n \\times k$ matrix. That is, for matrix multiplication to be applicable, the number of columns in the first matrix must equal the number of rows in the second matrix.\n",
+    "\n",
+    "Here is how matrix product is calculated: if we are calculating $AB = C$, then\n",
+    "\n",
+    "$$C_{i,j} = A_{i,0} \\cdot B_{0,j} + A_{i,1} \\cdot B_{1,j} + \\dotsb + A_{i,m-1} \\cdot B_{m-1,j} = \\sum_{t = 0}^{m-1} A_{i,t} \\cdot B_{t,j}$$\n",
+    "\n",
+    "Here is an example with relatively small matrices:\n",
+    "\n",
+    "$$\\begin{bmatrix}\n",
+    "    1 & 2 & 3 \\\\\n",
+    "    4 & 5 & 6\n",
+    "\\end{bmatrix}\n",
+    "\\begin{bmatrix}\n",
+    "    1 & 2 \\\\\n",
+    "    3 & 4 \\\\\n",
+    "    5 & 6\n",
+    "\\end{bmatrix}\n",
+    "=\n",
+    "\\begin{bmatrix}\n",
+    "    (1 \\cdot 1) + (2 \\cdot 3) + (3 \\cdot 5) & (1 \\cdot 2) + (2 \\cdot 4) + (3 \\cdot 6) \\\\\n",
+    "    (4 \\cdot 1) + (5 \\cdot 3) + (6 \\cdot 5) & (4 \\cdot 2) + (5 \\cdot 4) + (6 \\cdot 6)\n",
+    "\\end{bmatrix}\n",
+    "=\n",
+    "\\begin{bmatrix}\n",
+    "    22 & 28 \\\\\n",
+    "    49 & 64\n",
+    "\\end{bmatrix}$$\n",
+    "\n",
+    "Note that matrix multiplication is **not commutative:** $AB$ rarely equals $BA$. That being said, here are the properties it does have:\n",
```

Comment:
It seems better to start with the properties that is does have: 
```suggestion
    "Matrix multiplication has the following properties:\n",
```
(and make "Note that matrix multiplication is not commutative: 𝐴𝐵 rarely equals 𝐵𝐴." the last entry in the list of properties)